client/src/components/Overview/ImageShowcase.jsx

```diff
@@ -0,0 +1,17 @@
+import { StyledImgShowcase } from '../styles/Overview/ImageShowcase.styled';
+
+const ImageShowcase = ({productStyle}) => {
+  return (
+    <StyledImgShowcase>
+      {
+        productStyle.results[0].photos.map((photo, i) => {
+          return ( i === 0 &&
```

Comment:
Is this hard-coded to one image? This will need to be dynamic in the future

Reply:
Yes, right now is hard coded. I am working on dynamic loading it now